frontend/server/src/main/java/org/pytorch/serve/metrics/Metric.java

```diff
@@ -159,4 +172,28 @@ public String toString() {
         sb.append(",timestamp:").append(timestamp);
         return sb.toString();
     }
+
+    public static class telemetryMetricsBuilder {
```

Comment:
Why need this duplicate code class telemetryMetricsBuilder? Telemetry metric is just a metric which has no any difference with Metric. Existing Metric is able to support it completely.